tests/test_sdl_persistent_storage_attributes.ts

```diff
@@ -13,3 +13,14 @@ tap.test("SDL: Persistent Storage Manifest", async t => {
 
   t.matchSnapshot(result, "Manifest matches expected result");
 });
+
+tap.test("SDL: Persistent Storage with class 'ram' must have 'persistent' set to false", async t => {
```

Comment:
there is jest now ready to be used ;)

Reply:
Yea I knowww, I didn't want to spend time refactoring this since it was in the same file of a related test. I'll remake this file as a spec file.